index.js

```diff
@@ -226,35 +277,35 @@
      * @param {Function} callback Required to be called by DataTables. Used to update display
      */
     _requestData: function(tableParams, callback) {
-      this._updateColumnSortOrdering(tableParams);
```

Comment:
It looks like _updateColumnSortOrdering isn't being called anywhere anymore.  Is that intentional?  And if so, shouldn't we delete that method?